### PR TITLE
Fix Excel errors for numeric cells with spaces

### PR DESCRIPTION
**Pull request by vboctor**
Fixes #19927

### CODE REVIEW

**Comment by atrol** [comment]
Not sure if it's a good idea to change data when exporting.
I would expect that I get **exactly** the same data when exporting/importing. 
Let's also wait for feedback to https://www.mantisbt.org/bugs/view.php?id=19927#c51058

**Comment by FavioGalvis** [comment]
I am not having this issue, tryed to duplicate #19927 unsuccesfully. 
Also tested the code and this not have any repercutions on my reports, so I suppose a trim on report will not do any harm

:+1:

**Comment by vboctor** [comment]
I've updated the fix based on the discussion on the associated MantisBT issue.  Please test and confirm that everything is OK.

File: core/excel_api.php [hunk 1]
```diff
@@ -185,6 +190,12 @@ function excel_prepare_string( $p_value ) {
  * @return string
  */
 function excel_get_cell( $p_value, $p_type, array $p_attributes = array() ) {
+	if ( !is_int( $p_value ) ) {
+		$t_value = str_replace( array( '&', "\n", '<', '>' ), array( '&amp;', '&#10;', '&lt;', '&gt;' ), trim( $p_value ) );
```
Comment (atrol) [inline core/excel_api.php]: Is there still a need for the trim?
As mentioned before, I would expect that I get exactly the same data when exporting/importing.

**Comment by vboctor** [comment]
> Is there still a need for the trim?

@atrol I removed the trim.

**Comment by atrol** [comment]
now +1

**Pull request merged**